pkgs/tools/security/pinentry-mac/default.nix

```diff
@@ -17,6 +17,10 @@ stdenv.mkDerivation rec {
     mv build/Release/pinentry-mac.app $out/Applications
   '';
 
+  passthru = {
+    binaryPath = "Applications/pinentry-mac.app/Contents/MacOS/pinentry-mac";
```

Comment:
Well, I just thought that altering the original `pinentry` and `pinentry_qt5`, and then whichever new pinentries we get just because of one single macOS weirdness is not justified.

Do you really think we should do that?